models/stg_lever__opportunity.sql

```diff
@@ -22,27 +22,27 @@ fields as (
 final as (
     
     select 
-        _fivetran_synced,
+        cast(_fivetran_synced as {{ dbt_utils.type_timestamp() }}) as _fivetran_synced,
         cast(archived_at as {{ dbt_utils.type_timestamp() }}) as archived_at,
         archived_reason_id,
         contact as contact_id,
         cast(created_at as {{ dbt_utils.type_timestamp() }}) as created_at,
         data_protection_contact_allowed as is_data_protection_contact_allowed, 
-        data_protection_contact_expires_at,
+        cast(data_protection_contact_expires_at as {{ dbt_utils.type_timestamp() }}) as data_protection_contact_expires_at,
         data_protection_store_allowed as is_data_protection_store_allowed,
-        data_protection_store_expires_at,
+        cast(data_protection_store_expires_at as {{ dbt_utils.type_timestamp() }}) as data_protection_store_expires_at,
         headline as contact_headline,
         id as opportunity_id, 
         is_anonymized,
-        last_advanced_at,
-        last_interaction_at,
+        cast(last_advanced_at as {{ dbt_utils.type_timestamp() }}) as last_advanced_at,
+        cast(last_interaction_at as {{ dbt_utils.type_timestamp() }}) as last_interaction_at,
         location as contact_location,
         name as contact_name,
         origin,
         owner_id as owner_user_id,
-        snoozed_until,
+        cast(snoozed_until as {{ dbt_utils.type_timestamp() }}) as snoozed_until_at,
```

Comment:
I am suspicious that we should not cast this a timestamp. Since the original field is not `snoozed_until_at` I get the idea that this may not be a timestamp. Further we shouldn't change the name. I would remove the cast from this field.
```suggestion
        snoozed_until,
```

Reply:
Gotcha. I just noticed it was a timestamp in the macro so I wasn't sure, and then I knew we had specific formats for timestamp columns. But this would have more downstream impact. I'll leave it alone for now

`{"name": "snoozed_until", "datatype": dbt_utils.type_timestamp()},`